01_Exercises/LAB_03.py

```diff
@@ -0,0 +1,171 @@
+import os
+
+
+def do_list_integers(random_nr):
+    """
+    :param: must be a random number or a string number.
+    :return: a list of integers;
+    :note: The function will fail if the string is not numeric;
+    """
+    random_nr = str(random_nr)
+    random_list = list(random_nr)
+    all_letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't',
+                   'u', 'v', 'w', 'x', 'y', 'z']
+
+    for num in random_list:
+        if num in all_letters:
```

Comment:
Ar trebui sa explorezi invers solutia, inca pot pune `_!@@#!)(*)(*)(&%%^` si nu o sa crape validarea. Daca stii exact ce caractere ai voie sa ai e mai usor sa le acoperi pe alea:
`if num not in all_numbers` ....
Unde `all_numbers` sunt stringurile ['0', '1' ... '9']. Pe langa asta programul zice numere intregi deci trebui sa poti accepta si `-` ca si prefix.

Reply:
Nu e mai simplu sa adaug toate simbolurile?

Reply:
Defapt ai dreptate.

Reply:
Da am modificat acum si are mai mult sens asa.